corehq/apps/translations/app_translations.py

```diff
@@ -839,7 +839,7 @@ def has_translation(row_, langs):
             msgs.append((
                 messages.error,
                 _("You must provide at least one translation"
-                  " for the label '%s' in sheet '%s'") % (label, sheet.worksheet.title)
+                  " for the label '{0}' in sheet '{1}'").format(label, sheet.worksheet.title)
```

Comment:
should this be using named references as in the warning message?

Reply:
I went with the existing pattern in this file, although I prefer named references myself.

Reply:
well if we are changing should we do the right way then?

Reply:
Mostly trying to avoid scope creep here.  This code is 170 days old, so the transifex update has raised this warning and been ignored approximately 170 / 7 = 24 times.  At the risk of sounding like I don't care, if 23 previous times a developer chose to ignore this warning, I don't feel a lot of pressure to make this code perfect.  (I realize that people might take issue with that statement.)  Also, changing these string literals will require re-doing the translations in transifex, and I don't know if there is any particular frequency with which these get updated.

tl;dr - I'd rather leave this code better than I found it, today, than commit to a larger scope of work that I'm not able to get to anytime soon.  (Definitely not discouraging anyone else from making those changes though.)

Reply:
Sure nick. That sounds reasonable. Apologies if that comment felt like i was pushing for it.